frontend/src/components/ImageCropper/ImageCropper.module.scss

```
@@ -0,0 +1,37 @@
+@import "../../style/common.scss";
+.mainWrapper {
+  text-align: center;
+  height: 100vh;
+  width: 100vw;
+  position: fixed;
+  z-index: 100;
+  top: 0;
+  left: 0;
+  background-color: rgba($color: #000000, $alpha: 0.8);
+  overflow-y: scroll;
+}
+.ReactCrop {
```

```suggestion
.reactCrop {
```
Maintain consistency in variable naming style.